src/test/mir-opt/const-promotion-extern-static/rustc.BAR.PromoteTemps.diff

```diff
@@ -0,0 +1,65 @@
+  // MIR for `BAR`
+  // source = MirSource { instance: Item(DefId(0:6 ~ const_promotion_extern_static[317d]::BAR[0])), promoted: None }
```

Comment:
Can't we print this the way we print `GlobalId`? It's kind of ugly.

Reply:
`MirSource` only has an `InstanceDef` not an `Instance`, making this a bit harder. I can do a best effort variant.